### DISCUSSION TITLE
Unable to access the course on DIKSHA App

### DISCUSSION THREAD

**Discussion by Saras-Gautam** [Sunbird-Ed/Community#473] [Bugs] [answered]
The Punjab state team is facing issues with viewing the following
course: do_313748953265135616115 And I have tried to fix the issue but there is already a existing bug in the course due to which my fixes are not working on mobile phones and when that bug will get resolved then only the issue of mobile will get resolved.

**Comment by swayangjit** [comment]
@Saras-Gautam  Can you please mention the issue here ?

**Reply by Saras-Gautam** [reply]
Issue in this course is that the front page of the particular course is not working in any of mobile devices but able to access only in laptops.

**Comment by swayangjit** [comment]
@Saras-Gautam  The information provided in the thread is not sufficient to provide assistance on the issue that you have reported. Can you please provide additional detail so that the community can help you out with resolving the issue. Some examples of additional details are below

1. Issue is being faced in which version of Sunbird ED?
2. Is this issue faced only with a specific content or all content of the content type?
3. Is it being faced in all the mobile devices or a specific mobile device ?
4. Is it happening for a specific version of mobile OS ?
5. In which version of ED mobile app is this issue replicated?
6. Share the logs and the analysis done so far by your team and call out what specific help you need.
7. In the debug build are you seeing any error

**Reply by Saras-Gautam** [reply]
1:-version 5.1.0
2:-only specific content [do_313748953265135616115]
3:- all mobile devices.
4:- Android & IOS devices
5:- all versions.
6:-tried to update another image and uploaded in the portal but still not able to be seen in mobile devices.

**Reply by swayangjit** [reply]
@Saras-Gautam  Can you please elaborately explain the issue as I am not getting what is the issue? You can also post a video of the issue to explain.

**Reply by vinod303** [reply, marked as answer]
@Saras-Gautam since you are saying the issue is observed only in mobile, debug this issue further using the debug build and then post your findings here.
